great_expectations/checkpoint/actions.py

```diff
@@ -317,17 +315,6 @@ def _render_validation_result(
             validation_result_urls=validation_result_urls,
         )
 
-    def _send_notifications_in_batches(self, blocks, payload, result):
-        text = blocks[0]["text"]["text"]
-        chunks, chunk_size = len(text), len(text) // 4
-        split_text = [
-            text[position : position + chunk_size] for position in range(0, chunks, chunk_size)
-        ]
-        for batch in split_text:
-            payload["text"] = batch
-            result = self._get_slack_result(payload)
-        return result
```

Comment:
Feels like an optimization we don't need - would be easy enough to add something like this if we need it later on